client/src/main/java/io/pravega/client/tables/KeyValueTableConfiguration.java

```diff
@@ -26,5 +28,11 @@
      * The number of Partitions for a Key-Value Table. This value cannot be adjusted after the Key-Value Table has been
      * created.
      */
+    @Getter
     private final int partitionCount;
+
+    public KeyValueTableConfiguration(int partitionCount) {
```

Comment:
Revert changes to this file. the `@Data` annotation creates getters and constructors.

Reply:
done.